parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -63,6 +63,13 @@ def setup_args(cls):
     def __init__(self, opt: Dict[str, Any]):
 
         super().__init__(opt)
+        # Validate problem buckets
```

Comment:
This check already exists in the superclass, `AbstractTurnAnnotationResultsCompiler`, but below it looks like you're moving the check to the subclasses - is there an advantage to this? (Do you have a case in which `none_all_good` is not a bucket? If so, maybe we can just move this check down to the top of the abstract class's `.compile_results()` method, because I'd like future subclasses of this class to still have this check)

Reply:
yea it's in my other pull request https://github.com/facebookresearch/ParlAI/pull/3946/files#diff-8edc807d80efc9567c466a08f1732192ad2cf8f6300f478e607ccf69e8936a1bR21